backend/pkg/database/vulnerability.go

```diff
@@ -137,6 +139,11 @@ func (v *VulnerabilityTableHandler) Create(vul *Vulnerability) error {
 	if err := v.vulnerabilitiesTable.Create(vul).Error; err != nil {
 		return fmt.Errorf("failed to create vulnerability: %v", err)
 	}
+
+	if v.driverType == DBDriverTypePostgres {
+		refreshMaterializedViewsIfNeeded(v.db, v.driverType)
+	}
```

Comment:
no need for the wrapper "if v.driverType" here, its handled inside of refreshMaterializedViewsIfNeeded.